tests/backup/backup_test.go

```diff
@@ -7737,6 +7571,93 @@ func CreateBackupWithoutCheck(backupName string, clusterName string, bLocation s
 	log.FailOnError(err, "Failed to take backup with request -\n%v", bkpCreateRequest)
 }
 
+// backupSuccessCheck inspects backup task
+func backupSuccessCheck(backupName string, orgID string, retryDuration int, retryInterval int, ctx context.Context) (bool, error) {
+	var bkpUid string
+	backupDriver := Inst().Backup
+	if retryDuration == 0 {
+		retryDuration = maxWaitPeriodForBackupCompletionInMinutes
+	}
+	if retryInterval == 0 {
+		retryInterval = 30
+	}
+	backupSuccessCheck := func() (interface{}, bool, error) {
+		bkpUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+		log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
```

Comment:
return err here similar to return "", false, err

Reply:
Done